src/relocation.rs

```diff
@@ -211,6 +211,16 @@ mod overrides {
             self.set(src_prefix, src_prefix.name())
         }
 
+        /// Suppress relocations from the given source prefix.
+        pub fn suppress_and_parents(&mut self, mut src_prefix: Prefix<XorName>) {
```

Comment:
The name is slightly confusing, what about `suppress_self_and_parents` or something similar?